Encountify/ViewModels/NewLocationViewModel.cs

```diff
@@ -19,14 +18,16 @@ public NewLocationViewModel()
             SaveCommand = new Command(OnSave, ValidateSave);
             CancelCommand = new Command(OnCancel);
             SelectCommand = new Command(OnSelect);
-            this.PropertyChanged +=
+            PropertyChanged +=
                 (_, __) => SaveCommand.ChangeCanExecute();
         }
 
         private bool ValidateSave()
         {
-            return !String.IsNullOrWhiteSpace(name)
-                && !String.IsNullOrWhiteSpace(description);
+            return !string.IsNullOrWhiteSpace(name)
+                && !string.IsNullOrWhiteSpace(description)
+                && !string.IsNullOrWhiteSpace(longitude.ToString())
+                && !string.IsNullOrWhiteSpace(latitude.ToString());
```

Comment:
I don't think that lines 29-30 have any meaning, since they check if doubles converted to string are empty and I believe that it is always false.

Reply:
You're right, slightly misinterpreted this part. Also forgot to test it out at the end which showed that it does not do what I thought it would do. Will revert back to the old part. Thanks!